packages/engine-ui/src/components/BookingDetails.js

```diff
@@ -1,22 +1,49 @@
 import React from 'react'
 
 import Elements from './Elements'
+import styled from 'styled-components'
 
+const Paragraph = styled.p`
+  margin: 0;
+  margin-bottom: 2.5rem;
+  text-transform: capitalize;
+`
 const BookingDetails = ({ booking }) => {
-  if (!booking) return <p>Loading...</p>
+  const [address, setAddress] = React.useState()
+
+  const getAddressFromCoordinates = async ({ lon, lat }) => {
+    return await fetch(
+      `https://pelias.iteamdev.io/v1/reverse?point.lat=${lat}&point.lon=${lon}`
+    )
+      .then((res) => res.json())
+      .then(({ features }) => features[0].properties.label)
+  }
+
+  const getAddress = async ({ pickup, delivery }) => {
```

Comment:
I think the function name should rather be setAddress or similar, since it is not merely getting the address 🙂